src/component/Home-page-header/homepageheader.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+
+
+export default function HomePageHeader(){
+return (
+ <div className="flex flex-col place-items-center bg-pink">
+ 
+     
+       {/* <div className=" relative">  */}
+      {/* <div className="relative left-20 top-2   shadow-xl"> <img 
+       width="179px" height="282px"src="../images/header1.jpeg" alt="1" /></div>
+
+       <div className="relative left-60  -top-20 "><img width="250" height="311" src="../images/header3.jpeg" alt="3" /></div>
+       <div className="relative  left-20 -top-16 shadow-xl">
+           <img width="130" height="127"
+             src="../images/header2.jpeg" alt="2" /></div>
+       </div> */}
+
+  {/* <div className="static ml-8 mt-5
+           left-12 top-14 shadow-ll"> <img 
+       width="179px" height="282px" src="../images/header1.jpeg" alt="1" /></div>
+
+        <div className=" static  "><img width="250" height="3" src="../images/header3.jpeg" alt="3" /></div>
```

Comment:
why there are comments in the code? please don't submit a PR with comments in the code